support-frontend/assets/pages/new-contributions-landing/contributionsLandingInit.js

```diff
@@ -197,10 +212,13 @@ const init = (store: Store<State, Action, Function>) => {
 
   const state = store.getState();
 
-  initialisePaymentMethods(state, dispatch);
+  const contributionTypes = getContributionTypes(state);
+  dispatch(setContributionTypes(contributionTypes));
+
+  initialisePaymentMethods(state, dispatch, contributionTypes);
```

Comment:
`state.common.settings.contributionTypes` will not yet have been updated in the redux store, so the local `contributionTypes` value needs to be passed into `initialisePaymentMethods` and `selectInitialContributionTypeAndPaymentMethod`
This feels like an anti-pattern...

Reply:
haven't totally understood the issue yet - but wondering if keeping settings outside of the redux store would help? since they're immutable globals I've always thought maybe they shouldn't be in redux

Reply:
I was wondering this also! Ideally they'd be determined during initialisation and then passed in as props. I'll see what I can do

Reply:
I'm going to slap a `TODO` in here and try this in another PR as it's quite a big change, and we need the functionality in this PR asap